### ISSUE TITLE
Notify on failure

### ISSUE THREAD

**Issue by cpsauer** [Homebrew/homebrew-autoupdate#42]
First and foremost, thanks for releasing this. It's super duper handy for staying up-to-date by default :)
This is a feature request for what I think might be a common snag, based on my friends experience with it

Every so often, there's some random issue preventing brew from upgrading installed packages. Maybe `xcode-select --install needs rerunning`, or something needs a sudo password as in #40, or what have you. I usually don't catch those for a little while. It'd be great to have the notification feature tell you about failures, too. Or maybe just about new failures since the last run. Then I'd know about them and fix them immediately, letting your great auto-updater get back to work.

TL;DR: Notify about new failures upgrading so the user know when to fix them.

Thanks for your consideration!
Chris

(And apologies if it already does this somehow and I've misconfigured things.)

**Comment by DomT4** [comment]
Thanks for the idea & request! You're entirely correct to note that this currently isn't supported, but I don't disagree that it could & should be so there's not silent failures happening in the background if there's something persistent causing issues.

I've rolled this into https://github.com/Homebrew/homebrew-autoupdate/issues/43 since I've been mentally tracking a few todos on the notification side of things for a while, so I may as well make an actual tracking issue 😄.

**Comment by cpsauer** [comment]
Thanks, @DomT4. You're the man.
